duckduckgo/iOS
# Integrate SimulatorStatusMagic

Reviewer: Mia
Asana: https://app.asana.com/0/414235014887631/415242964614490/f
CC:

**Description**:

Sets the simulator status bar to clean defaults.

The Status Bar Magic framework is only added to the AppScreenshots target so will not be included in the released binary.

**Steps to test this PR**:
1. Run fastlane screenshots
2. Observe the resulting screenshots all have a clean status bar
3. Observe the simulator is reset to normal

###### Reviewer Checklist:


###### PR DRI Checklist:
